crates/sui-core/src/consensus_handler.rs

```diff
@@ -2264,14 +2262,11 @@ impl<C: CheckpointServiceNotify + Send + Sync> ConsensusHandler<C> {
                 SequencedConsensusTransactionKind::External(consensus_transaction) => {
                     match consensus_transaction.kind {
                         // === User transactions ===
-                        ConsensusTransactionKind::CertifiedTransaction(cert) => {
-                            // Safe because signatures are verified when consensus called into SuiTxValidator::validate_batch.
-                            let cert = VerifiedCertificate::new_unchecked(*cert);
-                            let transaction =
-                                VerifiedExecutableTransaction::new_from_certificate(cert);
-                            commit_handler_input.user_transactions.push(
-                                VerifiedExecutableTransactionWithAliases::no_aliases(transaction),
-                            );
+                        ConsensusTransactionKind::CertifiedTransaction(_) => {
```

Comment:
Let's revert non-test changes in this file. When validators will not deploy the change to remove QD handlers at the same time, so changes in handling of `CertifiedTransaction` can cause forks. We need to first disable accepting consensus blocks with `CertifiedTransaction`, after this change rolls out.